EXTENSION_POLICY.md

```diff
@@ -56,3 +56,51 @@ may be a single individual, but it is always preferred to have multiple reviewer
 In the event that the Extension PR author is a sponsoring maintainer and no other sponsoring maintainer
 is available, another maintainer may be enlisted to perform a minimal review for style and common C++
 anti-patterns. The Extension PR must still be approved by a non-maintainer reviewer.
+
+## Extension stability and security posture
+
+Every extension is expected to be tagged with a `status` and `security_posture` in its
+`envoy_cc_extension` rule.
+
+The `status` is one of:
```

Comment:
What is the relation between this and API versioning? Since I see, for example, xray tracer is WIP but the API is stable (v3, without alpha).

Reply:
Yeah, this is a source of a lot of confusion in my experience. API versioning only applies to API stability, not implementation. The two values can be different. I'll update with some clarification.

Reply:
Cool. And I think it will be great to have a way to render the extension (implementation) stability in the docs (please ignore if this is already the case).

Reply:
Yeah, the generated docs already have alpha indications in both the config API docs and the extension security posture status.